services/vidispine/chapter_test.go

```diff
@@ -0,0 +1,53 @@
+package vidispine
+
+import (
+	"testing"
+
+	"github.com/bcc-code/bcc-media-flows/services/vidispine/vsapi"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMergeTerseTimecodes(t *testing.T) {
+	terseA := map[string][]*vsapi.MetadataField{
+		"title": {
+			{
+				Start: "10@PAL",
+				End:   "20@PAL",
+			},
+		},
+	}
+
+	terseB := map[string][]*vsapi.MetadataField{
+		"title": {
+			{
+				Start: "15@PAL",
+				End:   "25@PAL",
+			},
+		},
+		"something_else": {
+			{
+				Start: "15@PAL",
+				End:   "25@PAL",
+			},
+		},
+	}
+
+	expected := map[string][]*vsapi.MetadataField{
+		"title": {
+			{
+				Start: "10@PAL",
+				End:   "25@PAL",
+			},
+		},
+		"something_else": {
+			{
+				Start: "10@PAL",
```

Comment:
We update all the metadatas in terseB to have the same in-out points.
This is how it works. It's used for already-split metadata (chapters)

BTW ive been thinking it could make sense to make a type-based distinction between the "raw" meta and "split" meta:
psuedo:
```
type Metadata {terse, stuff} 
type SplitMeta = Metadata
SplitByChapters(Metadata) returns SplitMeta
mergeTerseTimcodes(a, b SplitMeta)
```

but im not sure that's possible with go?